compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -129,6 +132,15 @@ unsafe fn configure_llvm(sess: &Session) {
 
     llvm::LLVMInitializePasses();
 
+    for plugin in &sess.opts.debugging_opts.llvm_plugins {
+        let path = Path::new(plugin);
+        let res = DynamicLibrary::open(path);
+        match res {
+            Ok(_) => debug!("configure_llvm: {}", plugin),
```

Comment:
nit: tracing already includes information about functions and modules that the debug call is in. This should perhaps say more about what's being done here precisely:

```suggestion
            Ok(_) => debug!("LLVM plugin loaded {} ({})", path.display(), plugin),
```

Or something along those lines.

Reply:
Makes sense, I will update it in this case. But how about this section, is it outdated? https://rustc-dev-guide.rust-lang.org/compiler-debugging.html#logging-etiquette-and-conventions

`A loosely followed convention is to use debug!("foo(...)") at the start of a function foo and debug!("foo: ...") within the function.`

Reply:
Huh, I wonder if that convention predates the use of tracing for logging. @oli-obk do you know?